server/mvc/controllers/BattlenetController.js

```diff
@@ -0,0 +1,19 @@
+var BattlenetUser = require("../models/User/BattlenetUser");
+var Promise = require("bluebird");
+var RegionProfile = require(process.env.ROOT + "/server/mvc/models/RegionProfile");
+
+var region = [
```

Comment:
Just as a sticky note, we should definitely make a region service or move the regions to config :)